dev-requirements.txt

```
@@ -1,16 +1,16 @@
 black==24.10.0; python_version > "3.8"
-black==24.8.0; python_version <= "3.8"
+black==24.10.0; python_version <= "3.8"
 isort; python_version < "3.8"
 isort==5.13.2; python_version >= "3.8"
 pre-commit; python_version <= "3.8"
 pre-commit==4.0.1; python_version > "3.8"
 flake8; python_version < "3.8"
 flake8==7.1.1; python_version >= "3.8"
-bandit==1.7.10; python_version <= "3.8"
+bandit==1.8.0; python_version <= "3.8"
```

```suggestion
bandit==1.7.10; python_version <= "3.8"
```
Reverting because this is the last version of bandit that supports 3.8.  We will be removing this line entirely soon, though.